inference-engine/src/mkldnn_plugin/emitters/jit_snippets_emitters.hpp

```diff
@@ -26,39 +46,56 @@ class KernelEmitter : public jit_emitter {
               const std::vector<size_t> &pool = {}, const std::vector<size_t> &gpr = {}) const override {
         emit_impl(in, out, pool, gpr, nullptr);
     }
+    jit_snippets_compile_args _jep;
 
 private:
     void emit_impl(const std::vector<size_t>& in,
                    const std::vector<size_t>& out,
                    const std::vector<size_t>& pool,
                    const std::vector<size_t>& gpr,
                    const MKLDNNPlugin::emitter_context *emit_context) const override {
-        auto tile_rank = in[0]; // count of tile dimensions
-        auto nparams = in[1];
+        const size_t tile_rank = in[0]; // count of tile dimensions
+        const size_t num_inputs = in[1];
+        const size_t num_outputs = in[2];
+        const size_t num_params = num_inputs + num_outputs;
         int reg64_tmp_start { 8 }; // R8, R9, R10, R11, R12, R13, R14, R15 inputs+outputs+1
-        Xbyak::Reg64 params   { dnnl::impl::cpu::x64::abi_param1 };
-        Xbyak::Reg64 schedule { dnnl::impl::cpu::x64::abi_param2 };
-        Xbyak::Reg64 offsets_ { dnnl::impl::cpu::x64::abi_param3 };
-        Xbyak::Reg64 offsets  { dnnl::impl::cpu::x64::abi_not_param1 };
+        Xbyak::Reg64 reg_indexes   { dnnl::impl::cpu::x64::abi_param1 };
```

Comment:
Minor: You have already used `using namespace Xbyak`

Reply:
Agree, removed Xbyak::